packages/quick_actions/quick_actions_ios/example/ios/RunnerTests/FLTQuickActionsPluginTests.m

```diff
@@ -0,0 +1,151 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import Flutter;
+@import quick_actions_ios;
+@import quick_actions_ios.Test;
+@import XCTest;
+#import <OCMock/OCMock.h>
+
+@interface FLTQuickActionsPluginTests : XCTestCase
+
+@end
+
+@implementation FLTQuickActionsPluginTests
+
+// A dummy `UIApplicationShortcutItem`.
+- (UIApplicationShortcutItem *)searchTheThingShortcutItem {
+  return [[UIApplicationShortcutItem alloc]
+           initWithType:@"SearchTheThing"
+         localizedTitle:@"Search the thing"
+      localizedSubtitle:nil
+                   icon:[UIApplicationShortcutIcon
+                            iconWithTemplateImageName:@"search_the_thing.png"]
+               userInfo:nil];
+}
```

Comment:
nits:
Since `UIApplicationShortcutItem` is immutable, maybe we can make it static/singleton to avoid the need of creating this object multiple times?

Reply:
I personally would rather duplicate code in tests if it makes it clearer what's being tested--clarity over performance.

Reply:
These are just data without complex logic, so I think we should be fine with the readability here. 

I read [a doc](https://abseil.io/tips/122) a while ago (recommended by @stuartmorgan), and it has an example of this kind of "free functions". 

```
BobberProto RecentCheapConcert() {
  return PARSE_TEXT_PROTO(R"(
      id: 21
      artist: "The Shouting Matches"
      when: "2016-08-24 20:30:21 -04:00"
      price_usd: 60)");
}
BobberProto PastExpensiveConcert() {
  return PARSE_TEXT_PROTO(R"(
      id: 17
      artist: "Beyonce"
      when: "2012-10-10 12:39:54 -04:00"
      price_usd: 200)");
}

TEST(BobberTest, UsesProtos) {
  Bobber bobber({PastExpensiveConcert(), RecentCheapConcert()});
  SomeCall();
  EXPECT_THAT(bobber.MostRecent(), EqualsProto(RecentCheapConcert()));
}
```
> Moving the initialization into free functions makes it clear that there is no hidden dataflow. Well chosen names for the helpers mean that you can likely review the test for correctness without even scrolling up to see the details of the helper.

Reply:
I agree with that, I don't like little utility classes or helpers in tests.

Reply:
Thinking about it again, maybe we should just return a new instance for simplicity/readability, instead of using static/singleton since it's complicating things. What do you think?